pkgs/applications/backup/timeshift/minimal.nix

```
@@ -0,0 +1,13 @@
+{ callPackage
+, timeshift-wrapper ? (callPackage ./wrapper.nix { })
```

```suggestion
, timeshift-wrapper
```
Please move that to top-level.